### PR TITLE
feat: job to manage the edge-lite cluster

### PR DESCRIPTION
**Pull request by kuisathaverat**
## What does this PR do?

* [chore: delete obsolete jobs](https://github.com/elastic/apm-pipeline-library/pull/1755/commits/5953ebd49f219821ee354b3bf4e7351ea124a09c)
* [chore: remove schedule of the update job](https://github.com/elastic/apm-pipeline-library/pull/1755/commits/050254de822d48753f36a6787d5def9d9ebaddb8)
* [feat: job to anage the edge-lite cluster](https://github.com/elastic/apm-pipeline-library/pull/1755/commits/3d7e28f0da1576eb8430e203420ae928971e2a48)

### CODE REVIEW

**Review by v1v — CHANGES_REQUESTED** [review]
`little` or `lite`?

File: .ci/jobDSL/jobs/apm-ci/apm-shared/oblt-test-env/edge-litle.groovy [hunk 1]
```diff
@@ -15,13 +15,14 @@
 // specific language governing permissions and limitations
 // under the License.
 
-pipelineJob("apm-shared/oblt-test-env/create-oblt-cluster") {
-  displayName('Create oblt cluster')
-  description('Job to create a oblt cluster.')
+pipelineJob("apm-shared/oblt-test-env/edge-litle-oblt-cluster") {
+  displayName("edge-litle-oblt cluster")
+  description("Job to create a edge-litle- oblt cluster every day.")
```
Comment (kuisathaverat) [inline .ci/jobDSL/jobs/apm-ci/apm-shared/oblt-test-env/edge-litle.groovy:20]:
```suggestion
pipelineJob("apm-shared/oblt-test-env/edge-lite-oblt-cluster") {
  displayName("edge-litle-oblt cluster")
  description("Job to create a edge-lite- oblt cluster every day.")
```
```suggestion
pipelineJob("apm-shared/oblt-test-env/edge-litle-oblt-cluster") {
  displayName("edge-litle-oblt cluster")
  description("Job to create a edge-litle- oblt cluster every day.")
```

File: .ci/jobDSL/jobs/apm-ci/apm-shared/oblt-test-env/edge-litle.groovy [hunk 2]
```diff
@@ -45,7 +46,7 @@ pipelineJob("apm-shared/oblt-test-env/create-oblt-cluster") {
         }
       }
       lightweight(false)
-      scriptPath(".ci/create.groovy")
+      scriptPath(".ci/edge-litle.groovy")
```
Comment (kuisathaverat) [inline .ci/jobDSL/jobs/apm-ci/apm-shared/oblt-test-env/edge-litle.groovy:49]:
```suggestion
      scriptPath(".ci/edge-lite.groovy")
```

File: .ci/jobDSL/jobs/apm-ci/apm-shared/oblt-test-env/edge-litle.groovy [hunk 3]
```diff
@@ -15,13 +15,14 @@
 // specific language governing permissions and limitations
 // under the License.
 
-pipelineJob("apm-shared/oblt-test-env/create-oblt-cluster") {
-  displayName('Create oblt cluster')
-  description('Job to create a oblt cluster.')
+pipelineJob("apm-shared/oblt-test-env/edge-lite-oblt-cluster") {
+  displayName("edge-litle-oblt cluster")
+  description("Job to create a edge-lite- oblt cluster every day.")
```
Comment (kuisathaverat) [inline .ci/jobDSL/jobs/apm-ci/apm-shared/oblt-test-env/edge-litle.groovy:20]:
```suggestion
pipelineJob("apm-shared/oblt-test-env/edge-lite-oblt-cluster") {
  displayName("edge-lite-oblt cluster")
  description("Job to create a edge-lite- oblt cluster every day.")
```

File: .ci/jobDSL/jobs/apm-ci/apm-shared/oblt-test-env/edge-litle.groovy [hunk 4]
```diff
@@ -15,13 +15,14 @@
 // specific language governing permissions and limitations
 // under the License.
 
-pipelineJob("apm-shared/oblt-test-env/create-oblt-cluster") {
-  displayName('Create oblt cluster')
-  description('Job to create a oblt cluster.')
+pipelineJob("apm-shared/oblt-test-env/edge-lite-oblt-cluster") {
+  displayName("edge-lite-oblt cluster")
+  description("Job to create a edge-lite- oblt cluster every day.")
```
Comment (kuisathaverat) [inline .ci/jobDSL/jobs/apm-ci/apm-shared/oblt-test-env/edge-litle.groovy:20]:
```suggestion
  description("Job to create a edge-lite-oblt cluster every day.")
```

**Review by v1v — APPROVED** [review]
LGTM.

I just renamed one of the files that was still pointing to `litle`

**Pull request merged**